datahub-web-react/src/app/auth/LogIn.tsx

```diff
@@ -128,6 +137,14 @@ export const LogIn: React.VFC<LogInProps> = () => {
                             }}
                         </Form.Item>
                     </Form>
+                    <Divider style={{ backgroundColor: 'white' }} />
```

Comment:
Nitpick: A few inline styles here. As you know, we try to avoid those to keep the structure code clean